docs/reference.md

```diff
@@ -240,3 +251,12 @@ Fired in case of error, can be at each request.
 Use it to customize HTTP error formatting for instance.
 
 Receives `request`, `response` and `error` parameters.
+
+
+### dispatch
```

Comment:
Another option: attach the route payload to the request, and let the users use the `request` hook instead of adding a new one?